### ISSUE TITLE
About torch.cat([encoder_hidden_state, ip_token,], dim=1)

### ISSUE THREAD

**Issue by lxd941213** [tencent-ailab/IP-Adapter#44]
Hello author, I have a stupid question. I read the code you provided and found that ip_token and encoder_hidden_state were cated. Is this different from the add described in the paper

**Comment by xiaohu2015** [comment]
@lxd941213 we split it in the attention processor https://github.com/tencent-ailab/IP-Adapter/blob/main/ip_adapter/attention_processor.py#L138

**Comment by lxd941213** [comment]
> @lxd941213 we split it in the attention processor https://github.com/tencent-ailab/IP-Adapter/blob/main/ip_adapter/attention_processor.py#L138

I see, thank you very much
